LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Craig Edwards and D++ Contributors
+   Copyright 2021-2023 Craig Edwards and D++ Contributors
```

Comment:
this is not how copyright works! Copyright doesnt have an end, so does not need an end date.

Reply:
(plus, we would need to ensure we went in every year and update it. This would get forgotten)

Reply:
Tbf that place in the LICENSE is afaik not even meant to be changed from the template, but I just went with it. Copyright naturally doesn't expire, well not in this way anyway, there's the Mickey Mouse law for that, but the place in time where something originated may play a role in it. Which is why yes, licenses are typically meant to be updated with current year, typically when you actually change something in there, so its obvious a particular piece of work has been last authored and therefore copyrighted at particular time and not a prior time already stated.